Task4/isbn_verifier.go

```diff
@@ -0,0 +1,55 @@
+package isbn
+
+func IsValidISBN(isbnExample string) bool {
+	var numArr []int
+
+	for i := 0; i < len(isbnExample); i++ {
+		switch isbnExample[i] {
```

Comment:
This `switch ` here looks odd. We can convert string symbol to int. (for ex. use `strconv.Atoi(string(symb))`)